setup.py

```diff
@@ -1,14 +1,33 @@
+import os
 import setuptools
 
```

Comment:
I would suggest to use `from photosynthesis_metrics import __version__` as we do not use additional files with metadata.

Reply:
Discussed with @denproc offline and decided to leave the existing implementation since it is more conventional and robust one